jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -80,17 +82,12 @@ public Builder addEntry(Path sourceFile, AbsoluteUnixPath pathInContainer) {
      *     sourceFile}
      * @param permissions the file permissions on the container. If null, then default permissions
      *     are used (644 for files, 755 for directories)
-     * @param lastModified the modification time of the file
      * @return this
      * @see Builder#addEntry(Path, AbsoluteUnixPath)
      */
     public Builder addEntry(
-        Path sourceFile,
-        AbsoluteUnixPath pathInContainer,
-        @Nullable FilePermissions permissions,
-        long lastModified) {
-      layerEntries.add(new LayerEntry(sourceFile, pathInContainer, permissions, lastModified));
-      return this;
+        Path sourceFile, AbsoluteUnixPath pathInContainer, @Nullable FilePermissions permissions) {
```

Comment:
This leaves us

`.addEntry(Path, AbsoluteUnixPath, @Nullable FilePermissions)`
`.addEntry(Path, AbsoluteUnixPath, @Nullable FilePermissions, Instant)`

where `@Nullable` means using the default. OTOH, you removed `@Nullable` from `LayerEntry`, leaving us

`LayerEntry(Path, AbsoluteUnixPath)`
`LayerEntry(Path, AbsoluteUnixPath, FilePermissions)`
`LayerEntry(Path, AbsoluteUnixPath, FilePermissions, Instant)`.

That is, the pattern is not consistent, so how about removing `@Nullable` and add `addEntry(Path, AbsoluteUnixPath)`? One downside of removing `@Nullable` is that the user will continue to pass a null after upgrading jib-core and get into trouble, but since we are not GA, I think this is fine.

That said, we should be conscious to have a consistent pattern involving `@Nullable` for default values.